pkgs/applications/networking/instant-messengers/webex/default.nix

```
@@ -0,0 +1,112 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, libGL
+, libX11
+, libXdamage
+, libXfixes
+, libXcomposite
+, libXrandr
+, cups
+, libxkbcommon
+, libsecret
+, libxcb
+, xcbutilwm
+, xcbutilrenderutil
+, xcbutilimage
+, xcbutilkeysyms
+, libdrm
+, libxshmfence
+, fontconfig
+, mesa
+, nss
+, wayland
+, udev
+, dbus
+, glib
+, libpulseaudio
+, alsaLib
+, at-spi2-core
+, at-spi2-atk
+, harfbuzz
+}:
+stdenv.mkDerivation {
+  pname = "webex";
+  version = "41.5.0.18815";
+
+  src = fetchurl {
+    # Official link: https://binaries.webex.com/WebexDesktop-Ubuntu-Official-Package/Webex.deb
+    # New versions get uploaded to same link, so lock url for this version in archive.org
+    url = "https://archive.org/download/webexhttpsbinaries.webex.comwebexdesktop-ubuntu-official-packagewebex.deb/Webex.deb";
```

why is that not the web.archive.org link?

---

Did I get the wrong link? I uploaded it to archive.org and this is the page I got the link from: https://archive.org/download/webexhttpsbinaries.webex.comwebexdesktop-ubuntu-official-packagewebex.deb
in that page I took the link for the "Webex.deb" file

---

```suggestion
    url = "https://web.archive.org/web/20210529010325/https://binaries.webex.com/WebexDesktop-Ubuntu-Official-Package/Webex.deb";
```

I think the link should be this.

---

thanks for the link, I updated it